packages/rest-api/src/utils/tokenAddressToToken.ts

```diff
@@ -1,19 +1,23 @@
+import { getAddress } from '@ethersproject/address'
+
 import { BRIDGE_MAP } from '../constants/bridgeMap'
 
 export const tokenAddressToToken = (chain: string, tokenAddress: string) => {
+  const address = getAddress(tokenAddress)
```

Comment:
1. Do we need to do this here? (note comment above where middleware will normalize & checksum addresses)

Reply:
Nope! Above middleware takes care of this, removed in https://github.com/synapsecns/sanguine/commit/30fd0c5fd08e6bfbc7ff9a2c71c09a87a9e4e634